R/latent_individual.R

```diff
@@ -71,6 +64,35 @@ epidist_family.epidist_latent_individual <- function(data, family = "lognormal",
   )
 }
 
+#' Define priors for the model
+#' 
+#' We provide suggested priors for the intercepts of the `meanlog` and `sdlog`
+#' linear predictors. These priors are weakly informative in that they prevent
+#' very large delays on the real scale. In particular:
+#'
+#' * `eta_meanlog ~ normal(2, 0.5)`
+#' * `eta_sdlog ~ normal(0, 0.5)`
+#' 
+#' Note that the link function used for `meanlog` is a constant, and the link
+#' function used for `sdlog` is a logarithm, such that the expectation of
+#' `sdlog` is the exponential of `eta_sdlog`.
+#' 
+#' To alter the priors for this model, we recommend first fitting using these
```

Comment:
I don't think we recommend first fitting do we? We are currently recommending using `get_prior` to `fn` to get the default priors?

Reply:
I don't think that `get_prior` as `fn` works. `get_prior` has the wrong inputs and will just return the default `brms` prior. What is better is `prior_summary` but that takes the fitted model as input. So there is something that needs to be changed here (suggest `prior_summary` not `get_prior`). At the moment I don't think there is any function which works as a drop in to `fn` that would get the prior without first fitting (perhaps this is something we can raise a `brms` issue for or somehow write our own wrapper to do it).

Reply:
Fixed `prior_summary` change in [f017a16](https://github.com/epinowcast/epidist/pull/95/commits/f017a160fa83e8eb44a26ba07feeef2dac388581).

Reply:
I don't think I do recommend this? I think we should suggest looking at the defaults first without fitting + any we specify and then if needing a more detailed view fitting and looking at prior_summary?

Reply:
> 1. Suggest looking at the defaults first without fitting + any we specify

> 2. If needing a more detailed view fitting and looking at `prior_summary`

Note that by including the defaults into `epidist_prior` I think would allow people not to have to do 2. and they could just do 1. (calling `epidist_prior`) to get all the prior information they would need to know to be able to edit the model.

(Basically having to call `get_prior`, see some priors on things, then also call `epidist_prior` and know that the `epidist_prior` ones superseed the `get_prior` ones is a confusing workflow which would be simplified by what I propsed in `epidist_priorB` above.)

Reply:
For the time being I'm happy to write out both of these options:

You can either look at the default priors, and look at our replacement priors (without fitting the model).

OR

If you have fitted the model, you can extract all the priors (with our defaults superseeding).

Reply:
I've updated the `roxygen2` to list both possible workflows.